daemon/emer-main.c

```diff
@@ -414,8 +419,13 @@ make_daemon (gint                argc,
     permissions_provider =
       emer_permissions_provider_new_full (config_file_path, NULL);
 
+  if (tracking_id_file_path != NULL)
+    machine_id_prov =
```

Comment:
Let's maintain the consistency with the rest of the file and spell out "provider" fully, unless you have a good reason to abbreviate it (for example, the longer name making legibility terrible somewhere else).

Reply:
I referred to `daemon/emer-daemon.c` line 1193. So I think it should be acceptable.
```
set_machine_id_provider (EmerDaemon            *self,
                         EmerMachineIdProvider *machine_id_prov)
```